interface/billing/apply_payments.php

```diff
@@ -47,16 +47,15 @@
 $eraname = '';
 $eracount = 0;
 /* Load dependencies only if we need them */
-if( ( isset($GLOBALS['portal_onsite_enable'])) || ($GLOBALS['portal_onsite_enable']) ){
+ if( !empty($GLOBALS['portal_onsite_enable']) ){
     /*  Addition of onsite portal patient notify of invoice and reformated invoice - sjpadgett 01/2017 */
     require_once("../../patient_portal/lib/portal_mail.inc");
     require_once("../../patient_portal/lib/appsql.class.php");
     
     function is_auth_portal( $pid = 0){
         if ($pData = sqlQuery("SELECT * FROM `patient_data` WHERE `pid` = ?", array($pid) )) {
-            if($pData['allow_patient_portal'] != "YES")
-                return false;
-                else return true;
+            if($pData['allow_patient_portal'] != "YES") return false;
```

Comment:
Would it be good to use the { } with the if statements. I am not sure how 7.1 and above are going to like this